channels/timed_set_test.go

```diff
@@ -23,47 +23,54 @@ func TestTimedSetMarshal(t *testing.T) {
 	var str struct {
 		Channels TimedSet
 	}
-	bytes, err := json.Marshal(str)
+	bytes, err := base.JSONMarshal(str)
 	assert.NoError(t, err, "Marshal")
-	goassert.Equals(t, string(bytes), `{"Channels":null}`)
+	assert.Equal(t, string(bytes), `{"Channels":null}`)
 
 	str.Channels = TimedSet{}
-	bytes, err = json.Marshal(str)
+	bytes, err = base.JSONMarshal(str)
 	assert.NoError(t, err, "Marshal")
 	goassert.Equals(t, string(bytes), `{"Channels":{}}`)
 
+	str.Channels = AtSequence(SetOf(t, "a"), 17)
+	bytes, err = base.JSONMarshal(str)
+	assert.NoError(t, err, "Marshal")
+	goassert.Equals(t, string(bytes), `{"Channels":{"a":17}}`)
```

Comment:
.

Added new test case for single property JSON can still be compared directly, as key order won't be affected here.